### PR TITLE
Add a simple web GUI

### PR DESCRIPTION
**Pull request by brendandburns**
@lavalamp @krousey

### CODE REVIEW

**Comment by krousey** [comment]
Screenshot or it didn't happen.

File: submit-queue/www/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          </md-content>
+        </div>
+      </div>
+      <!-- Angular Material Dependencies -->
+      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-animate.min.js"></script>
+      <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular-aria.min.js"></script>
+
+      <script src="https://ajax.googleapis.com/ajax/libs/angular_material/0.9.4/angular-material.min.js"></script>
+      <script src="http://localhost:8000/script.js"></script>
```
Comment (krousey) [inline submit-queue/www/index.html]: I know very little about Javascript/Angular, but this seems wrong.

**Comment by brendandburns** [comment]
screen shot 2015-09-09 at 11 08 04 am

**Comment by krousey** [comment]
Awesome. Looks good except for the reference to a script on localhost.

**Comment by brendandburns** [comment]
that's funny, it works on my machine...

(fixed)

**Comment by lavalamp** [comment]
Looks great, thanks.

**Pull request merged**